Video.js

```diff
@@ -24,6 +24,16 @@ export default class Video extends Component {
     this._root.setNativeProps(nativeProps);
   }
 
+  stringsOnlyObject(obj) {
+    const strObj = {};
+
+    Object.keys(obj).forEach(x => {
+      strObj[x] = obj[x].toString();
```

Comment:
This will set objects to `[object Object]` - do we want a check here for `typeof obj[x]` and to use `JSON.stringify` in those cases?

Reply:
Yes you're correct - I was kinda of unsure whether to leave it or do a "smarter" string conversion. After your review, I choose the latter. 👍 

Conversion is as follows:
-   Booleans, numbers, strings => `prototype.toString`
-   Objects and nulls => `JSON.stringify`'
-   Date => `prototype.toISOString`
-   `undefined` => returns empty string